src/es/components/molecules/buttons/Buttons.js

```diff
@@ -0,0 +1,115 @@
+// @ts-check
+import { Shadow } from '../../web-components-toolbox/src/es/components/prototypes/Shadow.js'
+
+
+/**
+* @export
+* @class Buttons
+* @type {CustomElementConstructor}
+*/
+export default class Buttons extends Shadow() {
+  constructor (options = {}, ...args) {
+    super({ importMetaUrl: import.meta.url, ...options }, ...args)
+  }
+
+  connectedCallback () {
+    if (this.shouldRenderCSS()) this.renderCSS()
+    if (this.shouldRenderHTML()) this.renderHTML()
+  }
+
+  disconnectedCallback () {}
+
+  /**
+   * evaluates if a render is necessary
+   *
+   * @return {boolean}
+   */
+  shouldRenderCSS () {
+    return !this.root.querySelector(`:host > style[_css], ${this.tagName} > style[_css]`)
+  }
+
+  /**
+   * evaluates if a render is necessary
+   *
+   * @return {boolean}
+   */
+  shouldRenderHTML () {
+    return !this.root.querySelector('.buttons-container')
+  }
+
+  /**
+   * renders the css
+   */
+  renderCSS () {
+    this.css = /* css */`
+      :host .buttons-container {
+        display: flex;
+        gap: 1rem;
+      }
+    `
+    return this.fetchTemplate()
+  }
+
+  /**
+   * fetches the template
+   */
+  fetchTemplate () {
+    /** @type {import("../../web-components-toolbox/src/es/components/prototypes/Shadow.js").fetchCSSParams[]} */
+    const styles = [
+      {
+        path: `${this.importMetaUrl}../../web-components-toolbox/src/css/reset.css`, // no variables for this reason no namespace
+        namespace: false
+      },
+      {
+        path: `${this.importMetaUrl}../../web-components-toolbox/src/css/style.css`, // apply namespace and fallback to allow overwriting on deeper level
+        namespaceFallback: true
+      }
+    ]
+    switch (this.getAttribute('namespace')) {
+      case 'buttons-default-':
+        return this.fetchCSS([{
+          path: `${this.importMetaUrl}./default-/default-.css`, // apply namespace since it is specific and no fallback
```

Comment:
could be commented out, since the file is empty

Reply:
As long you don't use the namespace, you don't load the file. Instead of commenting out, I would prefer to delete the code and its associated files.